ads/_a4a-config.js

```diff
@@ -84,3 +84,21 @@ export const signingServerURLs = {
   'cloudflare': 'https://amp.cloudflare.com/amp-ad-verifying-keyset.json',
   'cloudflare-dev': 'https://amp.cloudflare.com/amp-ad-verifying-keyset-dev.json',
 };
+
+/** @type {!../extensions/amp-a4a/0.1/refresh-manager.RefreshConfig} */
+const DEFAULT_REFRESH_CONFIG = {
```

Comment:
RefreshManager pulls in the config on its own. The `DEFAULT_REFRESH_CONFIG` object is there for convenience, in case other networks want to use the same sort of configuration DoubleClick is using. I can take it or leave it, though.